joblib/numpy_pickle.py

```diff
@@ -307,25 +314,23 @@ def __init__(self, filename, file_handle, mmap_mode=None):
             np = None
         self.np = np
 
-    def _open_pickle(self, file_handle):
-        return file_handle
-
     def load_build(self):
-        """ This method is called to set the state of a newly created
-            object.
+        """Called to set the state of a newly created object.
 
-            We capture it to replace our place-holder objects,
-            NDArrayWrapper, by the array we are interested in. We
-            replace them directly in the stack of pickler.
+        We capture it to replace our place-holder objects,
+        NDArrayWrapper, by the array we are interested in. We
```

Comment:
add NumpyArrayWrapper or NDArrayWrapper (backward-compatibility for joblib <= 0.9)